.github/workflows/ci.yml

```diff
@@ -55,22 +148,19 @@ jobs:
         with:
           name: tests-report-junit
           path: |
-            **/build/test-results/test/*.xml
+            **/target/surefire-reports/*.xml
 
       - name: Generate an access token to trigger downstream repo
         uses: actions/create-github-app-token@2986852ad836768dfea7781f31828eb3e17990fa # v1.6.2
         id: generate_token
-        # test only in main and report ubuntu results only
-        if: github.ref == 'refs/heads/main' && matrix.os == 'ubuntu-latest'
```

Comment:
why did we get rid of `&& matrix.os == 'ubuntu-latest'` ?

Reply:
Because this uploading only runs in the context of the ubuntu job, so we don't need to check anymore. It used to be a matrix build, so we had to check. Now, we don't upload in the macOS build at all, so no need to check.